GhostNetwork.Profiles.Domain/SecuritySettings/ISecuritySettingStorage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace GhostNetwork.Profiles.SecuritySettings
+{
+    public interface ISecuritySettingStorage
+    {
+        Task UpdateAsync(Guid userId, SecuritySetting updatedSettings);
```

Comment:
You have userId as a parameters and UserId inside SecuritySetting. Do you use both of them?